app/assets/stylesheets/custom.scss

```diff
@@ -92,4 +92,19 @@ li {
 
 .btn-secondary {
   margin-top: 8px;
+}
+
+#error_explanation {
```

Comment:
Why not a class?

Reply:
It's not a class because devise set this id default for div for errors.